NoSoft-SA/nspack
# Feature/serialized stock movement logs moved by

Summary of this change:
Serialized Stock Movement/Status Logs
- Added a new column 'moved_by' to serialized_stock_movement_logs table
- Stock movements now logs the user to this table
- Stock movements now also logs the user along with the status

How to test this code via the UI:
- Run any application that does a stock move
- After move, check pallet/bin status history to see if the user was logged
-  After move, go to Development | Logging | Stock Movement Logs to see if the user was logged
